src/PBot.Tests.Integration/Helper.cs

```diff
@@ -5,27 +5,35 @@
 //copy pasted from Octokit.net
 public static class Helper
 {
-    static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
```

Comment:
OK, I don't agree, but if this is controlled by RepoStandards then I won't argue against it here. Is this enforced by a failure in the R# analysis in the build?